schemas/data/propensity.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/propensity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Propensity",
+  "type": "object",
+  "description":
+    "The propensity schema represents the likelihood of a certain action happening in a certain window of time. The propensity schema can host various predictive results e.g churn. This is a general-purpose schema that is designed to contain the results of any propensity scoring algorithm.",
+  "definitions": {
+    "propensity": {
+      "properties": {
+        "xdm:score": {
+          "title": "Score",
+          "type": "number",
+          "description":
+            "The score is a real number between 0 and 1 (both inclusive). A higher value indicates a higher likelihood of the action occurring."
+        },
+        "xdm:action": {
```

Comment:
From @kstreeter (before reformatting moved stuff around)
> It's a little weird that this property is called "action", but the value is expected to map to a metric. It certainly makes sense that this maps to a metric, but shouldn't we then just call the property metric?

Reply:
I can see at least two ways of modeling propensity:

1. the likelihood of an event occurring (within a certain time frame)
2. the likelihood of observing a certain metric reaching or exceeding a certain value

In both cases it makes sense to have a stronger pointer (URI) here. In the second case, we also need to define what the threshold is and what kind of threshold (above, below) it is.

Reply:
Probably more a @cdegroot-adobe then @jwen-adobe question.

Reply:
Although if I understood @cdegroot-adobe's comment in #110 correct, we should establish the use case before we introduce this property at all.